### DISCUSSION TITLE
Discount codes via dropdown

### DISCUSSION THREAD

**Discussion by Schneleed** [liedekef/events-made-easy#479] [Ideas]
I would like to request a feature where I can use a dropdown list to let users select one specific discount code.
It would help to implement different age levels for example based on a simple dropdown selection in multiseat events especially.
There should be the option to name a code like child, adult, senior and add a respective value to it and display one dropdown in a booking Form where one of these three option can be selected.

**Comment by liedekef** [comment]
Discount codes are not meant to be used like this, so this request will never be realized. Create a custom field of type dropdown, set "extra price" to "yes", set as "field values" the price values for each (negative prices are accepted, so counting as discounts) and as "field tags" the names you want 'child, adult, ...'. The "field values" should be unique though, so watch out for that.

**Reply by Schneleed** [reply]
Ok understood, then I would like to request a feature where the mentioned custom field can be set once for a multiseat booking and being recognized for evrey seat booked. Basically adopting the setting from discounts "calculation for every booked seat" to the mentioned custom fields so that there is a real multi compatibility as otherwise the complexity using multiple dynamic form templates gets pretty intransparent to manage.

**Reply by liedekef** [reply]
The field I mentioned has no notion of data of other fields. Fields/values that rely on info from other fields are the reason I invented dynamic data. If nothing fits, feel free to code your own discount setup using a discount of type "code".

**Reply by liedekef** [reply]
> Ok understood, then I would like to request a feature where the mentioned custom field can be set once for a multiseat booking and being recognized for evrey seat booked. Basically adopting the setting from discounts "calculation for every booked seat" to the mentioned custom fields so that there is a real multi compatibility as otherwise the complexity using multiple dynamic form templates gets pretty intransparent to manage.

btw: I thought a little bit more aabout this: the current version allows extra costs (fixed or percentage) per payment method, so maybe that helps?
Next to that: you can already have a different price per seat category, is that not what you want too?
